packages/ui/VirtualList/tests/VirtualGridList-specs.js

```diff
@@ -228,68 +262,319 @@ describe('VirtualGridList', () => {
 			act(() => myScrollTo({position: {y: 100}, animate: false}));
 		});
 
-		describe('scroll events', () => {
-			test('should call onScrollStart once', () => {
-				render(
-					<VirtualGridList
-						cbScrollTo={getScrollTo}
-						clientSize={clientSize}
-						dataSize={dataSize}
-						itemRenderer={renderItem}
-						itemSize={itemSize}
-						onScrollStart={handlerOnScrollStart}
-					/>
-				);
+		test('should scroll to the given align with scrollTo', (done) => {
+			const onScrollStop = handlerOnScrollStop(done, () => {
+				const expected = Math.ceil(dataSize / 7 ) * (itemSize.minHeight + 3) - clientSize.clientHeight; // 1280/180 = 7
```

Comment:
Please fix all of the below.
```suggestion
				const expected = Math.ceil(dataSize / 7) * (itemSize.minHeight + 3) - clientSize.clientHeight; // 1280/180 = 7
```

Reply:
I fixed.